docs/modules/monitoring/con-integrating-micrometer.adoc

```diff
@@ -0,0 +1,96 @@
+// file included in the following:
+//
+// assembly-monitoring-proxy.adoc
+
+[id='con-integrating-micrometer-{context}']
+= Integrating Micrometer
+Kroxylicious integrates with https://micrometer.io/docs[Micrometer] for gathering metrics.
+
+Micrometer provides a simple facade over instrumentation clients for popular observability systems, allowing you to instrument your JVM-based application code without vendor lock-in. 
+The following example shows how to define the `CommonTagsHook` and `StandardBindersHook` types to add a label to metrics and register a JVM metrics binder.
+
+.Example proxy configuration for Micrometer integration
+[source,yaml]
+----
+adminHttp:
+  endpoints:
+    prometheus: {}
+micrometer:
+  - type: "CommonTagsHook" # <1>
+    config:
+      commonTags:
+        zone: "euc-1a" # <2>
+  - type: "StandardBindersHook" # <3>
+    config:
+      binderNames:
+      - "JvmGcMetrics" # <4>
+----
+<1> Specifies the `CommonTagsHook` type to add common tags to all metrics.
+<2> Adds common tag zone `euc-1a` to all metrics in the global registry included with Micrometer, which appears as a label in Prometheus.
+<3> Specifies the `StandardBindersHook` type to register standard Micrometer binders.
+<4> Registers the `JvmGcMetrics` binder with the global registry.
+
+Prometheus is connected to the Micrometer global registry, so filters can record metrics against it as part of the Prometheus scrape data.
+
+Using the `curl localhost:9999/metrics` command shows metrics as follows:
```

Comment:
Given we don't specify the port in sample config here we should probably use the default port `9190`